### PR TITLE
Complete 4 - View a list of all articles

### PR DESCRIPTION
**Pull request by garysum88**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,11 +1,37 @@
...
+  
   return (
+
     <div className="App">
+        <Header />
+        <Navbar />
+        <LoginBar />
+        <Routes>
+                <Route path="/" element={<Home />}></Route>
+                <Route path="/articles/*" element={<Articles />}></Route>
```
Comment (Rosem10) [inline src/App.js:28]: You'll need to change this endpoint once you get to working on displaying a single article. We want it to show all articles upon receiving /articles, and show a whole, single article when we go to /articles/:article_id - at the moment, if the user went to an endpoint for a single article (eg: /articles/1), or even /articles/banana - they would still get the list of all articles.

File: src/components/Articles.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+    return (
+      <>
+
+
+      <h3>Articles</h3>
+
+
+
+      <ListArticles />
```
Comment (Rosem10) [inline src/components/Articles.jsx:26]: I think this second layer of nesting components is unnecessary, and is also going to confuse things. If you think about the useEffect happening in your ListArticles component, it's set to trigger whenever the page is refreshed, but that only refers to the list component, not the whole articles endpoint.

File: src/components/ListArticles.jsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+      <div className="ListArticles__container">
+
+        { articles.map((article)=> {
+          return (
+            <div key={article.article_id} className="ListArticles__item">
+            <h4>{article.title}</h4>
+            <p>Topic : {article.topic}</p>
+            <p>Author : {article.author}</p>
+            <p>Created at : {article.created_at}</p>
+            <p><em>{article.comment_count} comment{article.comment_count !==0 ? "s" : null} </em> |  {article.votes} vote{ article.vote===0 ? null : "s"}</p>
```
Comment (Rosem10) [inline src/components/ListArticles.jsx:45]: This logic to add the conditional "s" is really, really nice!

**Review by Rosem10 — CHANGES_REQUESTED** [review]

File: src/components/Articles.jsx (hunk 2 above)
Comment (garysum88) [reply src/components/Articles.jsx:26]: There will be other components in "Articles.js", for examples, category menu, sort by options etc.
Therefore several states (e.g. category selected). I would keep this.

**Pull request merged**